frontend/src/compliance_reporting/components/ComplianceReportingPage.js

```diff
@@ -334,6 +334,7 @@ const ComplianceReportingPage = (props) => {
             {
               Array.from({ length: new Date().getFullYear() - 2017 }, (_, index) => {
                 const year = new Date().getFullYear() - index + 1
+                if (year === 2023) return null
```

Comment:
Let's make this null out any years greater than or equal to 2023. I know the ticket says we've already handled 2024 and above but it makes more sense just to limit the year values here.

Reply:
@AlexZorkin resolved